addons/interactions/src/preset/argsEnhancers.ts

```diff
@@ -0,0 +1,26 @@
+import { Args, addons } from '@storybook/addons';
+import { AnyFramework, ArgsEnhancer } from '@storybook/csf';
+import { fn } from 'jest-mock';
+import { EVENTS } from '../constants';
+import { instrument } from '../instrument';
+
+const { action } = instrument({ action: fn }, { retain: true });
```

Comment:
It's called action because that'll be how it's labelled in the UI. E.g. `expect(action(onSubmit)).toHaveBeenCalled()`
Otherwise you'd get `fn(onSubmit)`. I'll add a comment to clarify.